### DISCUSSION TITLE
How can i set a custom header tittle?

### DISCUSSION THREAD

**Discussion by yosoyfreeman** [mmistakes/minimal-mistakes#3546] [Q&A] [answered]
Hi, i'm working on the hope page and i would like to have a custom tittle for the header. So, if my site name is YoSoyFreeman, i want to show a "Hi, I'm freeman" on the header image, but i cant find a way to set this. 
Thank you for the help.

**Comment by mmistakes** [comment, marked as answer]
The header overlay uses what is assigned as the page's title.
You could try adding `title: "Hi, I'm freeman"` to the home page's YAML front matter and see if that works.

**Reply by yosoyfreeman** [reply]
Thank you so much, That worked pretty well. There is a way to hide the page tittle on the browser tab for this page? So it shows "SiteTitle" instead of "PageTitle-SiteTtile" ?

PD: Thank you so much for all your awesome work and the gorgeous documentation. Is helping me a lot in the learning process.
